### PR TITLE
Feature/services

### PR DESCRIPTION
**Pull request by cescoferraro**
solves https://github.com/dnephin/dobi/issues/45

### CODE REVIEW

**Comment by cescoferraro** [comment]
I cannot see this conflict

```
cescoferraro@desktop: ~/code/go/src/github.com/dnephin/dobi on feature/services [!]
$ git mergetool

This message is displayed because 'merge.tool' is not configured.
See 'git mergetool --tool-help' or 'git help config' for more details.
'git mergetool' will now attempt to use one of the following tools:
meld opendiff kdiff3 tkdiff xxdiff tortoisemerge gvimdiff diffuse diffmerge ecmerge p4merge araxis bc3 codecompare vimdiff emerge
No files need merging
```

File: tasks/service/run.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+		Mode: swarm.ServiceMode{
+			Replicated: &swarm.ReplicatedService{
+				Replicas: &replicas,
+			}},
+		Annotations: swarm.Annotations{
+			Name: name,
+		},
+		TaskTemplate: swarm.TaskSpec{
+			ContainerSpec: swarm.ContainerSpec{
+				Image: imageName,
```
Comment (cescoferraro) [inline tasks/service/run.go]: this needs #67 to be merged, so I can add labels to swarm.ContainerSpec

**Comment by cescoferraro** [comment]
A database is a service. 
Building go binaries is a job. 
A web-server is a service, and  people expect to tail their logs. 
So a service should honor the interactive field of the embedded job.
The question is: Services could have multiple replicas, should we attach to all of them at the same time?
Or just a random one?

**Comment by dnephin** [comment]
Swarm services now support logs, so could support a `:logs` action. However services still don't support interactive, so I don't think we should allow interactive yet on services.

**Pull request closed without merging**